lib/path.js

```diff
@@ -1100,38 +1089,26 @@ const posix = {
           // For example: from='/'; to='/foo'
           return to.slice(toStart + i);
         }
-      } else if (fromLen > length) {
-        if (from.charCodeAt(fromStart + i) === CHAR_FORWARD_SLASH) {
-          // We get here if `to` is the exact base path for `from`.
-          // For example: from='/foo/bar/baz'; to='/foo/bar'
-          lastCommonSep = i;
-        } else if (i === 0) {
-          // We get here if `to` is the root.
-          // For example: from='/foo'; to='/'
-          lastCommonSep = 0;
-        }
+      } else if (fromLen > length &&
+                 from.charCodeAt(fromStart + i) === CHAR_FORWARD_SLASH) {
+        // We get here if `to` is the exact base path for `from`.
+        // For example: from='/foo/bar/baz'; to='/foo/bar'
+        lastCommonSep = i;
       }
     }
 
-    var out = '';
+    let out = '';
     // Generate the relative path based on the path difference between `to`
-    // and `from`
+    // and `from`.
     for (i = fromStart + lastCommonSep + 1; i <= fromEnd; ++i) {
       if (i === fromEnd || from.charCodeAt(i) === CHAR_FORWARD_SLASH) {
         out += out.length === 0 ? '..' : '/..';
       }
     }
 
-    toStart += lastCommonSep;
-
     // Lastly, append the rest of the destination (`to`) path that comes after
-    // the common path parts
-    if (out.length > 0)
-      return `${out}${to.slice(toStart)}`;
-
-    if (to.charCodeAt(toStart) === CHAR_FORWARD_SLASH)
-      ++toStart;
-    return to.slice(toStart);
+    // the common path parts.
```

Comment:
pre-existing, but the comment should be capitalized

Reply:
Shouldn't sentences start with a capital letter? `s/the/The/`?

Reply:
The comment starts in the line above with `Lastly` :-)